clang/test/Headers/float.c

```diff
@@ -223,8 +223,8 @@
   #ifndef NAN
     #error "Mandatory macro NAN is missing."
   #endif
-  // FIXME: the NAN diagnostic should only be issued once, not twice.
```

Comment:
The fixme shouldn't be removed but updated to say the infinity and nan diagnostics should only be issued once, not twice.